packages/slider/src/HandleController.ts

```diff
@@ -522,12 +548,17 @@ export class HandleController {
                     aria-label=${ifDefined(model.ariaLabel)}
                     aria-labelledby=${ariaLabelledBy}
                     aria-valuetext=${this.formattedValueForHandle(model)}
+                    aria-describedby="sliderDesc"
                     @change=${this.onInputChange}
                     @focus=${this.onInputFocus}
                     @blur=${this.onInputBlur}
                     @keydown=${this.onInputKeydown}
                     .model=${model}
                 />
+                <p id="sliderDesc" class="visually-hidden">
```

Comment:
This should be OK as just `display: none` as it is `describedby` content. Otherwise you get multiple stops of the screen reader cursor. This should be confirmed manually after the change, preferably in [assistivlabs](https://assistivlabs.com/) across more than one screen reader, as also in a unit test using accessibility snapshotting.

Probably OK to make it a `span` over a `p` as it's not visible, too.